### DISCUSSION TITLE
SciChart configuration for nextJS using app router, Error: 'Module has already been declared', mono-repo

### DISCUSSION THREAD

**Discussion by mxke42** [vercel/next.js#62816] [App Router]
There seems to be no documentation on how to configure scichart when using app router with nextJS, and I am running into an error.

I have followed a combination of [this tutorial](https://www.scichart.com/documentation/js/current/webframe.html#Tutorial%2001%20-%20Setting%20up%20a%20Project%20with%20SciChart.js.html), and additional setup for nextJS found [here](https://github.com/ABTSoftware/SciChart.JS.Examples/tree/master/Sandbox/demo-nextjs), specifically the next.config.js file, copy-files-from-to.json, and additions to package.json. But I continue to get the error: ‘Module has already been declared’. (first/second pics in PDF)

I have attached a photos inside the pdf, including my project structure, and a zip with the files to duplicate.

Steps to replicate:
1) extract files and add root package.json to a directory of your choice
2) from within this directory, use ‘yarn create next-app scichart-prototype’ (use set-up options from 4th image in PDF)
3) Replace root layout.tsx and page.tsx with files in zip, add SciChartDemo directory and add page.tsx from that (also in zip)
4) replace the next.config.mjs file with the next.config.mjs file from the zip
5) add the copy-files-from-to.json from the zip
6) replace the package.json file in scichart-prototype directory with file from the zip.
7) run ‘yarn install’ to add packages
8) run ‘turbo dev’
9) after visiting localhost:3000/ click the Sci Chart Demo link to see error.

Thanks for taking a look.
<img width="554" alt="error" src="https://github.com/vercel/next.js/assets/28335323/623cb78c-6217-4762-8d16-5c1ba6a7126e">
<img width="464" alt="logs" src="https://github.com/vercel/next.js/assets/28335323/9bfcdd9c-4489-416b-b52a-6f4ee36d9a84">
<img width="136" alt="mono repo project structure with app router " src="https://github.com/vercel/next.js/assets/28335323/162d7609-bcdd-4cea-a474-57b783a58c50">
<img width="447" alt="next app initialization" src="https://github.com/vercel/next.js/assets/28335323/a725df1a-ca2d-4bf9-a52d-9c78c2ab1000">
[sci-chart-forum.pdf](https://github.com/vercel/next.js/files/14483068/sci-chart-forum.pdf)
[standalone.zip](https://github.com/vercel/next.js/files/14483083/standalone.zip)

**Comment by t3rrey** [comment]
Also getting same error +1
